simplq/src/components/common/Modal/Modal.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import { makeStyles } from '@material-ui/core/styles';
+
+const style = makeStyles((theme) => ({
+  root: {
+    overflow: 'scroll',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  content: {
+    position: 'absolute',
+    top: '20%',
+    backgroundColor: '#fff',
+    borderRadius: '25px',
+    outline: 'none',
+    width: 400,
+    padding: 10,
+    [theme.breakpoints.down('sm')]: {
+      width: '95%',
+      margin: 10,
+    },
+  },
+}));
+
```

Comment:
Could you move the styles for the content class to SCSS? Since we're using SCSS throughout the project, adding different forms of styling at different places unless really required would set a bad precedent. Ideally, we would want to use SCSS for the root class too, but we can keep makeStyles for root since material-ui is known to have some issues with CSS.

Reply:
Moved to scss